manifest.json

```diff
@@ -7,14 +7,13 @@
     "url": "https://github.com/BIDS-Apps/CPAC",
     "source": "https://github.com/flywheel-apps/c-pac",
     "license": "Apache-2.0",
-    "flywheel": "0",
-    "version": "0.1.0-dev.13",
+    "version": "0.1.14_1.4.1",
     "custom": {
```

Comment:
Please add  the following to the ```custom``` key. 
```json
"flywheel": {"suite": "BIDS Apps"}
```

That will help keep these gears sorted in the UI.